### PR TITLE
fixes #725

### LINKED ISSUE #696: Payment with Electronic Transfer does not fit within frame
screen shot 2015-09-14 at 21 39 36

### LINKED ISSUE #713: Typo in the successfully booked message after checkout
It says "All you activities" instead of "All your activities". See screenshot below

screen shot 2015-09-23 at 16 13 27

### LINKED ISSUE #721: Payment disabled description is shown even if the payment is ok to proceed
screen shot 2015-09-25 at 07 30 57

### LINKED ISSUE #722: Payment handling fee reason tooltip has pointer at wrong location
screen shot 2015-09-25 at 07 32 54

### LINKED ISSUE #725: Checkout payment options are not clear
Currently the payment options are a drop down list and assume that the user knows that the list is interactive and more options are available. This should be changed to a radio button selection so it is clear to the user what payment options are available

### PR DESCRIPTION
**Pull request by wszarmach**
fixes #725 
fixes #722
fixes #721
fixes #713
fixes #696

### CODE REVIEW

File: schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/booking-via-jasify.js [hunk 1]
```diff
@@ -299,8 +299,8 @@
 
             jasDialogs.ok(translationTitle, translationBody, onOk);
             function onOk() {
-                var w = $window.innerWidth || 820;
-                var h = $window.innerHeight || 620;
+                var w = $window.width || 820;
+                var h = $window.height || 620;
```
Comment (krico) [inline schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/booking-via-jasify.js]: Can we test this on IE, Firefox and safari also?

**Comment by krico** [comment]
Good stuff!

File: schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/booking-via-jasify.js (hunk 1 above)
Comment (wszarmach) [inline schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/booking-via-jasify.js]: Checked on all three browsers on a pc and works ok

Comment (wszarmach) [inline schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/booking-via-jasify.js]: Odd that the page you linked doesnt mention height or width though.

**Pull request merged**